dnsapi/dns_hostingukraine.sh

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env sh
+
+# Author: Qvinti (Aleksandr Zaitsev) <qvinti.com@gmail.com>
+# Report Bugs here: https://github.com/Neilpang/acme.sh/issues/2683 or https://www.ukraine.com.ua/forum/domennie-imena/acmesh-dnsapi-dlya-hosting-Ukra.html
+# Will be called by acme.sh to add the txt record to https://www.ukraine.com.ua/ DNS api.
+# Hosting Ukraine API documentation: https://api.adm.tools/osnovnie-polozheniya/dostup-k-api/
+# Usage: ./acme.sh --issue -d yourdomain.com [-d '*.yourdomain.com'] --dns dns_hostingukraine
+# API endpoint:
+HostingUkraine_Api="https://adm.tools/api.php"
+# Your login:
+HostingUkraine_Login=""
+# Your api token:
+HostingUkraine_Token=""
```

Comment:
remove it

Reply:
I put this lines in a comment like others dns